aws/environments/jrgm.yml

```diff
@@ -6,11 +6,11 @@ ssl_certificate_arn: arn:aws:acm:us-west-2:927034868273:certificate/ef6bb380-b02
 
 ec2_instance_type: t2.medium
 ec2_volume_size: 24
-fxadev_git_version: docker-emailservice-ses-policy
-customs_docker_state: stopped
-auth_ip_profiling_recency: '72 hours'
-auth_docker_tag: feature.emailservice
-email_service_docker_tag: feature.dockerimage
+fxadev_git_version: docker-remove-obsolete-git-version
+email_service_docker_tag: v1.118.0
+
+# intentional leftover
+oauth_git_version: v1.117.0
```

Comment:
Put in my config for testing only. I've built a new fxadev-jrgm stack with no problems.